inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html

```diff
@@ -34,7 +34,7 @@
       <div class="row">
         <div class="col-md-12 summary" id="detailed-header-top">
           <div class="col-md-2">
-            <img class="profile-image img-circle" src="{{ record['positions'][0]['email'] | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
+            <img class="profile-image img-circle" src="{{ record.get('positions', [])[0].get('email', '') | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
```

Comment:
I guess if there are no `positions` this line will still fail as it will try to access index `0` of empty list.

We might need to add a bit extra code:

```jinja2
{% set email = '' %}
{% if  record.get('positions', []) %}
    {% if  len(record.get('positions')) > 0 %}
        {% set email = record['positions'][0].get('email', '') %}
    {% endif %}
{% endif %}
<img class="profile-image img-circle" src="{{ email | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
```